src/views/LayoutWithRouting.tsx

```diff
@@ -83,9 +92,21 @@ const RouterWrapper = () => {
 
 const MainContainer = styled.main`
   padding: 0 var(--global-horizontal-padding);
-
+  margin-top: ${TOP_NAVBAR_HEIGHT}px;
   @media screen and (min-width: ${breakpoints.medium}) {
     margin-left: ${SIDENAVBAR_WIDTH}px;
   }
 `
+
+export const Container = styled.div`
```

Comment:
Few points here:
1) If we want this to be shared between views, let's extract it to `components`. Views shouldn't import from other views. We can call it `ViewWrapper`
2) It doesn't need `absolute` position I think, `relative` should be enough. Matter of fact, since `MainContainer` already takes care of padding and margins, I believe `position: relative` is the only style that needs to be here. Given that maybe we don't need that to be a component but rather just make sure all the views have `position: relative` in their top-most container? Or maybe we can export something like `viewWrapperCss` that can be used in all the views? Let's discuss